integration-tests/resteasy-mutiny/pom.xml

```diff
@@ -43,17 +43,6 @@
             <artifactId>awaitility</artifactId>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <groupId>org.jboss.resteasy</groupId>
-            <artifactId>resteasy-client</artifactId>
-            <scope>test</scope>
-            <exclusions>
-                <exclusion>
-                    <groupId>commons-logging</groupId>
-                    <artifactId>commons-logging</artifactId>
-                </exclusion>
-            </exclusions>
-        </dependency>
```

Comment:
Note that this does not change anything as the dependency is in `quarkus-resteasy-mutiny-common` so it comes transitively.